src/lib/layouts/Aside/Aside.svelte

```diff
@@ -119,8 +113,8 @@
 				break;
 		}
 	};
-	let ww = 0,
-		sidebar = null;
+	let ww = 0;
```

Comment:
@Valexr please, let's use more meaningful name for the variable, `ww` is very bad name

Reply:
fixed